### PR TITLE
 A few new SLAC acronyms

### PR DESCRIPTION
**Pull request by womullan**

### CODE REVIEW

File: etc/glossarydefs.csv [hunk 1]
```diff
@@ -220,6 +222,7 @@ DAA,Dual Axis Actuator,TS,,,A
 DAC,Data Access Center,LSST DM,,,A
 DAF,data access framework,TS,Document-11936,,A
 DALI,Data Access Layer Interface (IVOA standard),VO,,,A
+DAPC,Departmental Appointments and Promotion Committee,OPS,,,A
```
Comment (drphilmarshall) [inline etc/glossarydefs.csv:225]:
```suggestion
DAPC,SLAC FPD Departmental Appointments and Promotion Committee,OPS,,,A

```

Comment (drphilmarshall) [reply etc/glossarydefs.csv:225]: I think this only exists in the SLAC FPD context so maybe include those here? OK to leave them out too I think, up to you.

**Review by drphilmarshall — APPROVED** [review]
Thanks Wil! Youcould add "SLAC FPD" before "DAPC" just to make teh context clear, perhaps. I left a suggestion for you to take or leave.

**Comment by plazas** [comment]
Should the Spanish version be updated as well when new terms are added, so it doesn’t go out too out of sync?

**Comment by womullan** [comment]
I would not be sure how to translate dome of these office names .. but in principle yes we should try to keep in synch --- I tend to add a couple a week ..

**Comment by womullan** [comment]
@plazas I could  add a function to the script to output terms with no translation

**Pull request merged**